common/persistence/sql/sqlplugin/interfaces.go

```diff
@@ -121,6 +121,7 @@ type (
 		DomainID   serialization.UUID
 		WorkflowID string
 		RunID      serialization.UUID
+		Size       int
```

Comment:
Add some comments regarding two semantics of this Filter?

Reply:
Updated